drake/systems/plants/approximateIKEIQPmex.cpp

```diff
@@ -17,7 +17,7 @@
 #define USE_EIQUADPROG_BACKUP 1
 
 #if USE_EIQUADPROG_BACKUP
-#include "eiquadprog.hpp"
+#include "eiquadprog.h"
```

Comment:
Given my uncertainty about whether my local build will match everyone else's and whether CI will agree, I was inclined to not mess with it.  Once we delint the `build/include` rule, I believe this will get fixed.